java/src/main/java/com/splunk/support/lambda/TracerProviderConfigurer.java

```diff
@@ -27,23 +28,24 @@
 
   private static final String OTEL_LAMBDA_LOG_LEVEL = "OTEL_LAMBDA_LOG_LEVEL";
 
-  @Override
-  public void configure(SdkTracerProviderBuilder sdkTracerProviderBuilder) {
-    if (!"DEBUG".equalsIgnoreCase(System.getenv(OTEL_LAMBDA_LOG_LEVEL))) {
-      return;
+  private static void maybeEnableLoggingExporter(
```

Comment:
This was a little confusing. I'd move it below the call site so that it reads a bit more like a story.